app/queries/filter_rewards.rb

```diff
@@ -0,0 +1,17 @@
+class FilterRewards
```

Comment:
I find this name more fitting for the Query object (if you add something else apart of filtering, you wold have to change the name of the class)
```suggestion
class RewardsQuery
```